ftest.sh

```diff
@@ -362,19 +367,50 @@ if ! ./launch.py -c -a -r -i -s -ts ${TEST_NODES} ${TEST_TAG_ARR[*]}; then
 else
     rc=0
 fi
+copy_files=\$(clush -N -w \"${TEST_NODES}\" ls /var/tmp/core.\*)
+if [ -n \"\$copy_files\" ]; then
+    clush ${CLUSH_ARGS[@]} -B -v -l \"${REMOTE_ACCT:-jenkins}\" -R ssh \
+        -w \"${TEST_NODES}\" --rcopy \$copy_files --dest .
+fi
 
 # get stacktraces for the core files
 if ls core.*; then
     # this really should be a debuginfo-install command but our systems lag
     # current releases
-    python_rpm=\$(rpm -q python)
-    python_debuginfo_rpm=\"\${python_rpm/-/-debuginfo-}\"
-
-    if ! rpm -q \$python_debuginfo_rpm; then
-        sudo yum --enablerepo=\*debug\* -y install \$python_debuginfo_rpm
-    fi
-    sudo yum -y install gdb
-    for file in core.*; do
+sudo bash -c \"cat <<\\\"EOF\\\" > /etc/yum.repos.d/CentOS-Debuginfo.repo
+[base-debuginfo]
+name=CentOS-7 - Debuginfo
+baseurl=http://debuginfo.centos.org/7/\\\$basearch/
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Debug-7
+enabled=0
+EOF\"
+    debug_pkgs=\"python glibc:glibc-debuginfo-common daos systemd ndctl
+                libpmem:pmdk-debuginfo mercury cart libfabric argobots\"
+    install_pkgs=( gdb )
+    for pkg in \$debug_pkgs; do
+        if [[ \$pkg = *:* ]]; then
+            debug_pkg=\${pkg#*:}
+            pkg=\${pkg%:*}
+        else
+            debug_pkg=\"\$pkg-debuginfo\"
+        fi
+        if ver=\"\$(rpm -q --qf %{evr} \$pkg)\"; then
+            debug_pkg+=\"-\$ver\"
+            install_pkgs+=( \$debug_pkg )
+        fi
+    done
+    sudo yum --enablerepo=\*debug\* -y install \${install_pkgs[@]}
+    for file in \$(ls core.daos_io_server.*[0-9]); do
```

Comment:
Would it make sense to move the collection of the core files into launch.py as to associate the core file with the test file execution as opposed to the test phase?  When we collect core files they get placed at the same level as the test file sub-directories, under the test phase artifact directory.  If the core file was in the test file sub-directory it would be easier to associate the core file with the test failure and also easier to bug as it would be included in the `all files in zip` shortcut.

Reply:
Done.